static/js/components/VideoPlayer_test.js

```diff
@@ -66,6 +66,7 @@ describe("VideoPlayer", () => {
       fluid:         sandbox.stub().returns(playerStub),
       width:         sandbox.stub(),
       height:        sandbox.stub(),
+      hotkeys:       sandbox.stub(),
```

Comment:
if `this.hotkeys(...)` is called within `onPlayerReady`, you can change the "uses videojs on mount with the right arguments" test as follows:
```javascript
        const hotkeysStub = sandbox.stub()
        args[2].call({
          enableTouchActivity: enableTouchActivityStub,
          on:                  onStub,
          tech_:               { hls: {} },
          hotkeys:             hotkeysStub
        })
        sinon.assert.calledWith(hotkeysStub, {
          volumeStep:                0.1,
          seekStep:                  5,
          enableModifiersForNumbers: false
        })
```